calico-cloud/compliance/enable-compliance.mdx

```diff
@@ -0,0 +1,42 @@
+---
+description: Enable compliance to configure reports to assess compliance for all assets in a Kubernetes cluster.
+---
+
+# Enable Compliance
+
+## Big picture
+
+Enabling compliance improves the cluster's compliance posture. It involves generating compliance reports for Kubernetes clusters based on archived flow and audit logs for Calico Enterprise and Kubernetes resources. The process includes components for snapshotting configurations, generating reports, managing jobs, providing APIs with RBAC, and benchmarking security.
```

Comment:
I'm not sure that first statement is true. Doesn't this just enable reporting?

Reply:
compliance include compliance reports, compliance snapshot and compliance benchmark. How able using "compliance features" instead @ctauchen